{"name": "test_call_api_with_error_status_codes[403-API调用失败：403 Client Error]", "status": "passed", "description": "测试各种HTTP错误状态码", "parameters": [{"name": "status_code", "value": "403"}, {"name": "expected_exception_msg", "value": "'API调用失败：403 Client Error'"}], "start": 1761614695532, "stop": 1761614695535, "uuid": "27cb0ea6-47f0-4b28-a46d-328da1487863", "historyId": "05750348ca0ea3d7d2184323ca71e749", "testCaseId": "0ba28f683659835500083ccde95cdb5e", "fullName": "testcases.test_error_response.TestCallApi#test_call_api_with_error_status_codes", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_error_response"}, {"name": "subSuite", "value": "TestCallApi"}, {"name": "host", "value": "DESKTOP-SQEMBVG"}, {"name": "thread", "value": "15580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_error_response"}]}